docs/reference/setup/install/docker.asciidoc

```diff
@@ -372,6 +372,10 @@ published ports with `--publish-all`, unless you are pinning one container per h
 
 . Use the `ES_JAVA_OPTS` environment variable to set heap size. For example, to
 use 16GB, use `-e ES_JAVA_OPTS="-Xms16g -Xmx16g"` with `docker run`.
++
+--
+NOTE: You still need to configure the heap size even if you are https://docs.docker.com/config/containers/resource_constraints/#limit-a-containers-access-to-memory[limiting memory access] to the container.
```

Comment:
Suggest linking to the "setting the heap size" page, and formatting for width:

```suggestion
NOTE: You still need to <<heap-size,configure the heap size>> even if you are
https://docs.docker.com/config/containers/resource_constraints/#limit-a-containers-access-to-memory[limiting
memory access] to the container.
```

Reply:
Thanks! Addressed in https://github.com/elastic/elasticsearch/pull/42754/commits/274917c804916c69a15fb9fcffffd7fb77aa5631